css/css-text/text-indent/text-indent-percentage-003.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang=en>
+<meta charset="utf-8">
+<title>CSS Text Test: text-indent percentage resolution basis</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="https://drafts.csswg.org/css-text-3/#text-indent-property">
+<meta name="flags" content="">
+<link rel="match" href="reference/text-indent-percentage-002-ref.html">
+<meta name="assert" content="Percentages in text-indent	refer to width of the element's content box">
+<style>
+section { position: absolute; }
+section, div {
+  border-right: 10px solid white;
+  margin-right: 10px;
+  padding-right: 10px;
+}
+div {
+  box-sizing: border-box;
+  width: 120px;
+}
+.test div { text-indent: 50%; color: red; overflow: hidden; } /* overflow:hidden should not make any difference, but it does in some browsers */
+.ref div { text-indent: 50px; }
```

Comment:
Could a test for `calc(px + %)` be added if it's not too much of a hassle? That's a case that gecko gets wrong.

Reply:
How is that: https://github.com/web-platform-tests/wpt/pull/11373/commits/d25a341c1e87e040ce092887ed89ef2156b6c189 ?